ledger/boxtxn_test.go

```diff
@@ -169,28 +169,28 @@ func TestBoxCreate(t *testing.T) {
 		}
 
 		dl.txn(adam.Args("check", "adam", "\x00\x00"))
-		dl.txgroup("box_create; assert", adam.Noted("one"), adam.Noted("two"))
+		dl.txgroup("logic eval error: assert failed", adam.Noted("one"), adam.Noted("two"))
 
 		bobo := call.Args("create", "bobo")
 		dl.txn(bobo, fmt.Sprintf("invalid Box reference %#x", "bobo"))
 		bobo.Boxes = []transactions.BoxRef{{Index: 0, Name: []byte("bobo")}}
 		dl.txn(bobo)
-		dl.txgroup("box_create; assert", bobo.Noted("one"), bobo.Noted("two"))
+		dl.txgroup("logic eval error: assert failed", bobo.Noted("one"), bobo.Noted("two"))
```

Comment:
> Is it worth adding a flag to DoubleLedger that it uses with its BlockEvaluator, that sets a flag checked in EvalContext to still call pcDetails?

Having a flag that is checked in `EvalContext` would also enable us to conditionally include opcodes if DeveloperAPI is enabled. One of the concerns with removing the opcodes from the message is that some developers currently use it to bubble up error messages directly from the program upon failure. Definitely an ugly hack but still something some people do and it would be nice not to break that.